docs/product-assembly_installing-dm-on-premise/src/main/asciidoc/ba/master-docinfo.xml

```diff
@@ -1,7 +1,10 @@
 <productname>{PRODUCT}</productname>
```

Comment:
Structure looks pretty good to me. You might be getting to this eventually, but the only thing I'd say is that the Preface (which we'll hopefully be able to git rid of eventually) should comprise only the purpose of and any prerequisites for the whole assembly. It's *supposed* to follow the title immediately in normal circumstances (when we're not bound by ccutil), and precede the rest of the modules. In this case, looks like your preface has a lot of stuff that can just be moved to the existing Introduction that follows the Preface. So just a matter of keeping that preface to the why and for whom of the doc, plus any high-level prereqs.

For an example, see the now-completed (under SME and Christy review) [GDT assembly](http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/). Again, far from perfect, and still pending feedback from Christy and a peer, but that's my understanding from the templates.

Reply:
Yeah, I was saving that for when I have more content, but thanks.